using-timescaledb/compression.md

```diff
@@ -296,14 +296,99 @@ SELECT compress_chunk(i) from show_chunks('conditions', newer_than, older_than)
 
 ### Decompressing Chunks [](decompress-chunks)
 
-Next we will walk through what to do in the event that you need to backfill or
-update data that lives in a compressed chunk.
+In order to decompress individual chunks, you can run a `decompress_chunk`
+command in much the same way you can manually compress an individual chunk:
 
-TimescaleDB does not support inserts or updates into a compressed chunk. To insert
-or update data, we must first decompress the target chunks. But before we can do
-that, we need to turn off our compression policy. Otherwise that policy will attempt
-to re-compress the chunks that we are currently working on (not the desired result).
-To accomplish this we first find the job_id of the policy using:
+``` sql
+SELECT decompress_chunk('_timescaledb_internal._hyper_2_2_chunk');
+```
+
+Similar to above, you can also decompress a set of chunks based on a
+time range by first looking up this set of chunks via `show_chunks`:
+
+``` sql
+SELECT decompress_chunk(i) from show_chunks('conditions', newer_than, older_than) i;
+```
+
+Or if you want to have more precise matching constraints, including that you
+are using space partitioning (e.g., based on `device_id`):
+
+``` sql
+SELECT tableoid::regclass FROM metrics
+  WHERE time = '2000-01-01' AND device_id = 1
+  GROUP BY tableoid;
```

Comment:
Note that this will only work if the device and point already exist, if they don't exist this won't work. I'd love to have a better way to get this for prospective points, if there's a function I should know about that will do that so I can write this better for the next version of the backfilling stuff?

Reply:
Yes this assumes that you didnt delete tuples from those chunks/ you don't have empty chunks